cli/cfncluster/cfncluster.py

```diff
@@ -489,15 +453,11 @@ def delete(args):
             sys.stdout.flush()
         if status == 'DELETE_FAILED':
             logger.info('Cluster did not delete successfully. Run \'cluster delete %s\' again' % stack)
```

Comment:
(not related to boto3) typo 'cfncluster'

Reply:
fixed